### DISCUSSION TITLE
WSL AWS CLI설치 중 unzip 명령이 에러가 발생합니다.

### DISCUSSION THREAD

**Discussion by daecheol93** [tedilabs/fastcampus-devops#7] [Q&A] [answered]
WSL에서 CLI 설치중 unzip이 되질 않습니다..ㅠㅠ
이건 어떻게 해야할까요..?

![image](https://user-images.githubusercontent.com/82934184/131703183-a90d90bd-c50c-45ad-8e85-78cd043980f5.png)
![image](https://user-images.githubusercontent.com/82934184/131703044-eff555b4-5198-48ad-8bcb-d4e56b448340.png)

**Comment by posquit0** [comment]
`file awscliv2.zip` 결과 첨부해주실 수 있을까요? 혹시 아직 다운로드 중은 아닌지 확인해보시는 것도 좋을 듯 합니다.

**Comment by daecheol93** [comment, marked as answer]
Curl로 받아오는 과정에서 문제가 생겼었나 봅니다. 그냥 해당 url 클릭하여 다운로드 받고, mv로 옮겨와서 unzip하니 되었어요!

2021년 9월 3일 (금) 오전 1:02, Byungjin Park (Claud) ***@***.***>님이
작성:

**Comment by nojahoon** [comment]
저도 동일현상이 었었는데요, pdf에 있는 command
curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.2.16.zip" -o "awscliv2.zip" 에

위와 같은 것을 복사할 때  linux-가  linux로 잘못 복사되는 것 같아서 문제가 생겼던 것 같아요.(-가 ㄱ같은 모형으로 바뀌네요)

(curl "https://awscli.amazonaws.com/awscli-exe-linuxx86_64-2.2.16.zip" -o "awscliv2.zip")

**Reply by posquit0** [reply]
정보 공유해주셔서 감사합니다! ㅎㅎ
